src/disco/utils.js

```diff
@@ -0,0 +1,7 @@
+import { sanitizeHTML as baseSanitizeHTML } from 'core/utils';
+import purify from 'disco/purify';
+
+
+export const sanitizeHTML = (text, allowTags = [], _purify = purify) => {
```

Comment:
This could be renamed to reflect that the default behaviour is to modify all links to open in an new tab.

Reply:
Although I suppose that then risks that future usage would try and import this from here and modify the existing purify instance.

Reply:
I can totally do this actually, but then I'll remove the `_purify` parameter. I only need to come up with a good name 😬

Reply:
`sanitizeHTMLNewTabLinks`?

Reply:
`sanitizeHTMLWithNewTabLinks`?

Reply:
> sanitizeHTMLWithNewTabLinks?

It's long-winded but very clear. 👍

Reply:
⛵️